packages/components/date-picker-overlay/types/date-picker-overlay.d.ts

```diff
@@ -0,0 +1,33 @@
+export interface TSDatePickerOverlayHTMLAttributes {
+	dir?: string;
+
+	translations?: object;
+
+	firstDay?: number;
+
+	pageDate?: string;
+
+	selectedDate?: string;
+
+	required?: boolean;
+
+	disabledDateCheck?: function;
+
+}
```

Comment:
Not related to this PR but we should also think about a way to have the internal component like these in a separate section probably and exclude them from some stuff like type generation.

Reply:
The easiest solution would be just to have a list of these components. Another solution is to add a JSDoc comment on top of the class and process it in our doc generator so we can filter later by this comment.
Anyway, I think we don't have that many internal components and I don't think that we will move some components between internal and public states very often, so we can go with simple list.